### DISCUSSION TITLE
backups in case want to un-delete

### DISCUSSION THREAD

**Discussion by zmth** [wxMaxima-developers/wxmaxima#1587] [Q&A]
If i acidently or whatever delete some line in a notebook session and want to get it back is there any way to do that and if so how? i thought i remembered seeing something to that effect several months ago.

**Comment by gunterkoenigsmann** [comment]
Just place the cursor outside any editor cell and press Ctrl+Z: Each editor cell has its own undo functionality and the worksheet has an undo functionality, too.

**Comment by zmth** [comment]
though i guess if one has already completely deleted that error cell then that would not bring it back since it has been deleted or do you mean it would completely replace the whole notebook and all it's cells to the just prior one. By the way i usually have autosave  ON.
    On Saturday, September 25, 2021, 03:03:34 AM CDT, Gunter Königsman ***@***.***> wrote:  
 
 


Just place the cursor outside any editor cell and press Ctrl+Z: Each editor cell has its own undo functionality and the worksheet has an undo functionality, too.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by zmth** [reply]
unfortunately that did not work. Because of maximas bug of overwriting and garbling ones input changing ones input ,running two different input whatever u call it together,so it's impossible to get ones original back. Wish they would fix this horrible bug of maxima garbling and overwriting !
I loose atleast one input whatever you call it of many lines how humiliating it is to try and remember what i had before. I can't make copies every single time and i never know when maxima is going to screw up and garble again!
